x-pack/plugins/security_solution/public/management/components/console/components/console_code_block.tsx

```diff
@@ -5,7 +5,39 @@
  * 2.0.
  */
 
-import { EuiCode } from '@elastic/eui';
+import React, { memo, ReactNode } from 'react';
+import { EuiText, EuiTextColor } from '@elastic/eui';
 import { euiStyled } from '@kbn/kibana-react-plugin/common';
 
-export const ConsoleCodeBlock = euiStyled(EuiCode).attrs({ transparentBackground: true })``;
+export const ConsoleCodeBlock = memo<{
+  children: ReactNode;
+  inline?: boolean;
+  textColor?: 'default' | 'error' | 'success';
+  bold?: boolean;
+}>(({ children, inline = false, textColor = 'default', bold = false }) => {
+  const baseStyledComponent = inline ? EuiTextColor : EuiText;
+
+  const CodeBlock = euiStyled(baseStyledComponent).attrs({
+    transparentBackground: true,
+    size: 's',
+  })`{
+          color: ${(props) => {
+            if (textColor === 'error') {
+              return props.theme.eui.euiColorDanger;
+            } else if (textColor === 'success') {
+              return props.theme.eui.euiColorSuccessText;
+            } else {
+              return props.theme.eui.euiColorDarkestShade;
+            }
+          }} !important;
```

Comment:
Is the `!important` necessary?

Reply:
no, I updated here: https://github.com/elastic/kibana/pull/135756/commits/de5d7c15dfa1e0a98b94d602b4c183409a973e8d